Runtime/Scripts/Qonversion.cs

```diff
@@ -13,9 +13,29 @@ public class Qonversion : MonoBehaviour
         public delegate void OnOfferingsReceived(Offerings offerings, QonversionError error);
         public delegate void OnEligibilitiesReceived(Dictionary<string, Eligibility> eligibilities, QonversionError error);
 
+        /// <summary>
+        /// Delegate fires each time a promo purchase from the App Store happens.
+        /// Be sure you define a delegate for the event <see cref="PromoPurchasesReceived"/>. 
+        /// </summary>
```

Comment:
```suggestion
        /// Be sure you define a delegate for the event <see cref="PromoPurchasesReceived"/>. 
        /// Call StartPromoPurchase in case of your app is ready to start promo purchase. 
        /// Or store that delegate and call later when you need.
        /// </summary>
```